src/main/java/com/justcode/Main.java

```diff
@@ -0,0 +1,34 @@
+package com.justcode;
+
+import com.justcode.company.Company;
+import com.justcode.company.CompanyAccount;
+import com.justcode.company.employee.*;
+import com.justcode.company.exceptions.NotFoundAnyEmployeeException;
+
+public class Main {
+    static private CompanyAccount account = new CompanyAccount(new Company("JustCode"));
+
+    public static void main(String[] args) {
+        addTestEmployees();
+        account.printEmployeesName();
+        account.printEmployeesData(account.getSortedEmployeesBySalary(1500, 3500));
+        try {
+            account.printEmployeesData(account.getAllPermanentEmployee());
+        } catch (NotFoundAnyEmployeeException e) {
```

Comment:
NotFoundAnyEmployeeException can we put try-catch inside the method getAllPermanentEmployee